free-3.md

```diff
@@ -10,41 +10,98 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_Right now, without certain access and friends, it is difficult to play
+different types of card games competitively (Magic, Pokemon, Netrunner,
+etc.) because someone needs to buy the cards, which are not cheap, and
+practice playing, which requires someone else. Currently, most people
+choose not to play, for financial reasons or for lack of people to play
+with. Hopefully, this DSL would get them excited enough that they could 
+seek out people to play with because they would already know the game._
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_These games have wide range of the types of cards that that can have. So,
+having a DSL allows the user to have variable rules and variable cards. It
```

Comment:
I think this is the biggest point. There is a huge range of how these games work, and even within the games there are often huge variances in how it works. I think that writing a one-size fits all DSL might be a bad choice, and instead you'd want to write a good DSL for just one of them (but maybe you can find a way to abstract some aspects out to a library that can be used by multiple DSLs).

Also, I'd argue that these card games are already DSLs :smiley: